api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -63,6 +63,21 @@ message SessionMetadata {
   string PrivateKeyPolicy = 3 [(gogoproto.jsontag) = "private_key_policy,omitempty"];
 }
 
+// The kind of user a given username refers to. Usernames should always refer to
+// a valid cluster user (even if temporary, e.g. SSO), but may be Machine ID
+// bot users.
+enum UserKind {
+  // Indicates a legacy cluster emitting events without a defined user kind.
+  USER_KIND_UNSPECIFIED = 0;
+
+  // Indicates the user associated with this event is human, either created
+  // locally or via SSO.
+  USER_KIND_HUMAN = 1;
```

Comment:
Is "human" really the correct angle here? Could we make it a potentially more useful "login"/"tctl"/"bot"?